concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -39,6 +40,12 @@ prepare_environment() {
   cf_graphite_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.graphite.version "${cf_manifest_dir}/055-graphite.yml")
   cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "${cf_manifest_dir}/055-graphite.yml")
 
+  if ${COMMIT_VERIFICATION} ; then
```

Comment:
That's true, but also true of all the other variables at the moment.  I think it's better to be consistent.